dev/MRTCore/mrt/Microsoft.Windows.ApplicationModel.Resources/src/ResourceContext.cpp

```diff
@@ -97,6 +97,10 @@ void ResourceContext::Apply()
             winrt::check_hresult(MrmSetQualifier(m_resourceContext, eachValue.Key().c_str(), eachValue.Value().c_str()));
         }
     }
+    if (!ApplicationLanguages::PrimaryLanguageOverride().empty())
```

Comment:
>PrimaryLanguageOverride [](http://example.com/codeflow?start=31&length=23)

explicit language context should trump PrimaryLanguageOverride

Reply:
With W.G.PrimaryLanguageOverride set, newly created ResourceContext takes PrimaryLanguageOverride into account, i.e. does not override to default language.

Reply:
e.g. if default language is "en-US", PrimaryLanguageOverride is "de-DE", and ResourceContext["Lanauge"]="fr-FR", what's the language that will take effect? I'd think "fr-FR"

Reply:
according to [this logic](https://github.com/microsoft/WindowsAppSDK/blob/main/dev/MRTCore/mrt/Microsoft.Windows.ApplicationModel.Resources/src/ResourceContext.cpp#L43-L73), no, PrimaryLanguageOverride should take precedence over ResourceContext["Language"]. GetLangugageContext returns Windows::Globalization::ApplicationLanguages::Languages() which is affected if W.G.PrimaryLanguageOverride is set. This behavior remains the same with this change

Reply:
I think in that logic ResourceContext["Language"] still takes precedence. It would be strange to the caller that an explicit request is not honored

Reply:
Got it now. You're right. Fixed by latest change. Now explicit set of context Language is not overridden by PrimaryLanguageOverride.

I tested with [this WinAppSDK sample](https://github.com/microsoft/WindowsAppSDK-Samples/blob/main/Samples/ResourceManagement/cpp/cpp-winui/winui_desktop_packaged_app_cpp/MainWindow.cpp).  Default language is PrimaryLanguageOverride - de-DE, but fetching resources with m_overrideResourceContext passed is fetching it-IT resources:

![image](https://github.com/microsoft/WindowsAppSDK/assets/57057282/5948fb44-88b9-41e4-be1b-35dea39d5449)